app/views/articles/show.html.erb

```diff
@@ -1,7 +1,7 @@
 <% title @article.title %>
 
 <%= render "shared/webcomponents_loader_script" %>
-<%= javascript_packs_with_chunks_tag "clipboardCopy", "webShare", "articlePage", defer: true %>
+<%= javascript_packs_with_chunks_tag "clipboardCopy", "webShare", "articlePage", "articleModerationTools", defer: true %>
```

Comment:
We could hide this behind a `user_signed_in?` flag because we don't need this for non-logged-in traffic.

Reply:
Done! 😄